redis-hbo-provider/src/main/java/com/facebook/presto/statistic/RedisPlanStatisticsProvider.java

```diff
@@ -93,11 +93,13 @@ public Map<PlanNodeWithHash, HistoricalPlanStatistics> getStats(List<PlanNodeWit
                         output.put(redisFutureEntry.getKey(), redisFutureEntry.getValue().get());
                     }
                     catch (Exception e) {
+                        // This exception will be handled by the RedisProviderApiStats which will increment metrics and not fail the query if stats not found
                         log.error(String.format("Error reading done Redis future fore key %s", redisFutureEntry.getKey().toString()));
```

Comment:
not part of this change, but typo here fore -> for.  but also, this exception and the below get logged in RedisProviderApiStats, so no need to log them here too.